lib/fastlane/plugin/phraseapp/actions/phraseapp_action.rb

```diff
@@ -0,0 +1,58 @@
+require 'fastlane/action'
+require_relative '../helper/phraseapp_helper'
+
+module Fastlane
+  module Actions
+    class PhraseappAction < Action
+      def self.run(params)
+        check_phraseapp_binary
+
+        UI.message("PhraseApp is setup! Running binary...")
+
+        command = [
+          "cd #{File.expand_path('.').shellescape}",
+          "phraseapp #{params[:command]}"
+        ].join(" && ")
+
+        Fastlane::Actions.sh_control_output(
```

Comment:
I think it is a better solution to use the phraseapp gem because with your solution, you will have to install the CLI before using this action.
What do you think ? 
https://github.com/phrase/phraseapp-ruby

Reply:
> it is a pure API client for programmaticly using our features inside your own applications and does not support the push or pull commands for locale files 

They had a cli gem at some point but it got deprecated 😞

Reply:
Hummm I should have read it more carefully ... :(